pandas/core/frame.py

```diff
@@ -8116,9 +8117,13 @@ def _series_round(s, decimals):
     # ----------------------------------------------------------------------
     # Statistical methods, etc.
 
-    def corr(self, method="pearson", min_periods=1) -> "DataFrame":
+    def corr(
+        self,
+        method: Union[str, Callable[[np.ndarray, np.ndarray], np.float64]] = "pearson",
+        min_periods: Optional[int] = 1, axis: Union[str, int] = 0
```

Comment:
Running black on this file should fix the linting, the other CI issues seem unrelated

https://pandas.pydata.org/docs/dev/development/contributing.html#python-pep8-black

Reply:
Thanks @dsaxton It seems to be an issue with black. Had to modify a docstring completely unrelated to changes in this commit. Don't know why black didn't catch it earlier though.